src/lib/pio.pl

```diff
@@ -109,6 +109,9 @@
 - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - */
 
 phrase_to_file(GRBody, File) :-
-        setup_call_cleanup(open(File, write, Stream),
+        phrase_to_file(GRBody, File).
```

Comment:
This seems not correct, please test this definition of `phrase_to_file/2`.

Reply:
Ops, my mistake, sorry